src/Pages/WarehouseDetailPage/WarehouseDetailPage.js

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from "react";
+import "./WarehouseDetailPage.scss";
+import PageWrapper from "../../Components/PageWrapper/PageWrapper";
+import { useNavigate, useParams } from "react-router-dom";
+import axios from "axios";
+
+const WarehouseDetailPage = () => {
+  const { warehouseName, warehouseId } = useParams();
+  const navigate = useNavigate();
+  const [warehouse, setWarehouse] = useState({});
+  const [loading, setLoading] = useState(false);
+
+  const handleEditWarehouseNavigation = () => {
+    // Navigate to the warehouse Edit page with the warehouse ID
+    navigate(`/warehouse/${warehouseId}/edit`);
+  };
+
+  useEffect(() => {
+    // Fetch the warehouse data from the API
+    const getWarehouseDetailByWarehouseId = async () => {
+      setLoading(true);
+      try {
+        const { status, data } = await axios.get(
+          `${process.env.REACT_APP_BACKEND_URL}warehouses/${warehouseId}`
+        );
+        if (status === 200) setWarehouse(data);
+        setLoading(false);
+      } catch (error) {
+        console.error(error);
+      }
+    };
+    getWarehouseDetailByWarehouseId();
+  }, [warehouseName, warehouseId]);
+
+  return (
+    <PageWrapper
+      title={warehouseName}
+      handleButtonClick={handleEditWarehouseNavigation}
+      location={"Warehouse-details"}
+    >
+      {!loading ? (
+        <>
+          <section className="Warehouse-details__container">
+            <article className="Warehouse-details__infor--container address--padding">
+              <h2 className="Warehouse-details__infor--container-header">
+                WAREHOUSE ADDRESS:
+              </h2>
+              <p className="Warehouse-details__infor--container-text">
+                {warehouse.address}
+              </p>
+            </article>
+
+            <article className="Warehouse-details__infor">
+              <div className="Warehouse-details__tablet--divider" />
+              <article className="Warehouse-details__infor--container">
+                <h2 className="Warehouse-details__infor--container-header">
+                  CONTACT NAME:
+                </h2>
+                <p className="Warehouse-details__infor--container-text">
+                  {warehouse.warehouse_name}
```

Comment:
The details rendered are incorrect. This should be {warehouse.contact_name}. 

Your code renders:
<img width="818" alt="Screenshot 2024-09-21 at 11 05 46" src="https://github.com/user-attachments/assets/594ad799-fbe1-42b1-8a06-53016dd96734">

The code should render:
<img width="299" alt="Screenshot 2024-09-21 at 11 06 51" src="https://github.com/user-attachments/assets/3d2e4737-8db8-46bb-94fb-e07b2aec2e94">

Reply:
yeah that's right i will look into that thanks